Ginger/Ginger/Actions/ActionEditPages/WebServices/ActWebAPIModelEditPage.xaml.cs

```diff
@@ -272,5 +273,11 @@ private void MapOutputToDataSource(object sender, RoutedEventArgs e)
                 Reporter.ToUser(eUserMsgKey.MappedtoDataSourceError);
             }
         }
+
+        private void xViewAPIBtn_Click(object sender, RoutedEventArgs e)
+        {
+            APIModelPage mAPIEditPage = new APIModelPage(AAMB);
```

Comment:
don't use "FindAndReplace" only because it has similar behaver to what you need...
Use "Standalone" and just add it also to the ShowAsWindow switch case so it will be supported in page.